<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_listas</class>
 <widget class="QDialog" name="d_listas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTreeWidget" name="t_todo">
     <column>
      <property name="text">
       <string>TODO</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="b_quitar">
     <property name="text">
      <string>QUITAR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QTreeWidget" name="t_lista">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>CONTENIDO</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="b_crear">
     <property name="text">
      <string>CREAR</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="b_cancelar">
     <property name="text">
      <string>CANCELAR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cb_lista"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>                                                                                   Lista:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="b_agregar">
     <property name="text">
      <string>AGREGAR</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="b_eliminar">
     <property name="text">
      <string>ELIMINAR</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="b_seleccionar">
     <property name="text">
      <string>SELECCIONAR</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lin_crear"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
